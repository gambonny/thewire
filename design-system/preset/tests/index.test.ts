import Preset from '../src/index'

describe.concurrent('a test', () => {
  it('matches the snapshot', ({ expect }) => {
    expect.assertions(1)
    expect(Preset).toMatchSnapshot()
  })
})
