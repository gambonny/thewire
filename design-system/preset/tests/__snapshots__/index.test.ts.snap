// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`a test > matches the snapshot 1`] = `
{
  "name": "core",
  "theme": {
    "extend": {
      "tokens": {
        "colors": {
          "background": {
            "default": {
              "value": "{colors.gray.900}",
            },
            "muted": {
              "value": "{colors.gray.700}",
            },
            "overlay": {
              "value": "rgba(0, 0, 0, 0.5)",
            },
            "surface": {
              "value": "{colors.gray.800}",
            },
          },
          "border": {
            "default": {
              "value": "{colors.white}",
            },
            "subtle": {
              "value": "{colors.gray.600}",
            },
          },
          "cobalt": {
            "50": {
              "value": "#EBEDF9",
            },
          },
          "text": {
            "default": {
              "value": "{colors.white}",
            },
            "muted": {
              "value": "{colors.gray.400}",
            },
          },
        },
      },
    },
  },
}
`;
